packages/wonder-blocks-form/src/components/__tests__/text-area.test.tsx

```diff
@@ -977,8 +977,9 @@ describe("TextArea", () => {
                 expect(textArea).toHaveAttribute("aria-invalid", "false");
             });
 
-            it("should call the validate function when it is first rendered", async () => {
+            it("should call the validate function twice when it is first rendered (once on initialization, once after mount)", async () => {
                 // Arrange
+                // Act
                 const validate = jest.fn();
```

Comment:
**nit:** This mock would go in the `// Arrange` section instead.

For context: https://khanacademy.atlassian.net/wiki/spaces/ENG/pages/98402353/Unit+Testing+Best+Practices#Arrange-Act-Assert